### ISSUE TITLE
Implement & Test Our Architecture

### ISSUE THREAD

**Issue by BieniekAlexander** [ABieniek/metadataMaker#4]

**Comment by BieniekAlexander** [comment]
From what I understand, we're still trying to implement an architecture that reliably does paper classification with an abstract as input. The issues we're running into are the size of our dataset, the type of word/character embedding that we're using, and some other topics related to the architecture.

**Comment by DavidEnslevNyrnberg** [comment]
We don't really work with an embedding atm. just a Nx127(?) input space, for the character based input window

**Comment by BieniekAlexander** [comment]
> We don't really work with an embedding atm. just a Nx127(?) input space, for the character based input window

Right, we were looking into alternatives because the model that we had made wasn't decreasing in loss. It may have been the amount of data we have, but it also might have been that our embedding wasn't sophisticated enough. Now that we have more data, maybe we can try with our previous embedding.
